plugins/cilium-cni/types/types_test.go

```diff
@@ -248,10 +244,10 @@ func (t *CNITypesSuite) TestReadCNIConfAzurev2WithPlugins(c *check.C) {
       "azure": {
         "interface-name": "eth1"
       },
-      "ipam": {
-        "pre-allocate": 5
-      }
-    }
+	  "ipam": {
```

Comment:
Same as above, there are white-space changes here, please undo them.

Reply:
Yes, I noticed them as well after submitting the commit. Looks ok in my IDE though. Will check once again

Reply:
I am working on it. Will ping when its done

Reply:
@gandro, @nbusseneau fixed. Please, take a look.